js/events/drag.js

```diff
@@ -286,6 +286,10 @@ const DragEmitter = GestureEmitter.inherit({
             return false;
         }
 
+        if($prevTarget && $target.find($prevTarget).length) {
```

Comment:
Use closest instead of find because it works faster.
`$prevTarget.closest($target).length`
Also $prevTarget always exists because it is passed as $(result). So null ref check can be removed.